coral/src/app/features/topics/schema-request/TopicSchemaRequest.test.tsx

```diff
@@ -620,7 +620,7 @@ describe("TopicSchemaRequest", () => {
         environment: "1",
         remarks: "",
         schemafull: "{}",
-        topicName: "my-awesome-topic",
+        topicname: "my-awesome-topic",
```

Comment:
The inconsistent casing of the api params will always be something we will have micro-asdness about I think haha! Probably difficult to change at this point ^^